### DISCUSSION TITLE
Welcome to php-glfw Discussions!

### DISCUSSION THREAD

**Discussion by mario-deluna** [mario-deluna/php-glfw#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by odarcan** [comment]
Are you still here I asked some questions long time ago

**Reply by mario-deluna** [reply]
Hey Odarcan. I must have missed them, can you repeat or link?

**Reply by odarcan** [reply]
https://github.com/mario-deluna/php-glfw/discussions

https://github.com/mario-deluna/php-glfw/discussions/8

https://github.com/mario-deluna/php-glfw/discussions/10

**Reply by mario-deluna** [reply]
Thanks, i need to setup github in a way to get notifications..
Im currently in the middle of no where without a computer but i will try to provide specific answers / help soon ;)

In the meantime i really can recommend: https://learnopengl.com/Lighting/Basic-Lighting
That book applies almost 1 by 1 to PHP-GLFW.

specifically when it comes to shading, there is nothing special in php implementation.

**Comment by KingBes** [comment]
very perfec

**Comment by FluffyDiscord** [comment]
This is genius, i have never heard of this extension before
